client/my-sites/customer-home/cards/features/help-search/style.scss

```
@@ -37,9 +37,13 @@ $min_results_height: 180px;
 	.help-search__cta {
 		display: flex;
 		align-items: center;
+		width: 100%;
 		padding: 10px $card_padding_small;
 		color: var(--color-text);
 		line-height: 1;
+		font-size: $font-body;
+		font-weight: 400;
+		text-rendering: optimizeLegibility; /* match rest of inline help */
```

I don't see necessity of using `text-rendering`
Also, it's better to use `font-size: inherit;`, in this case, if somebody changes font size for all links in the list - it's necessary to do only in 1 place, instead of two.
```suggestion
		font-size: inherit;
```